frontend/containers/DocumentList/DocumentList.jsx

```diff
@@ -75,7 +75,7 @@ class DocumentList extends Component {
   constructor(props) {
     super(props)
 
-    this.state.selectedRows = {}
+    this.state = this.defaultLocalState()
```

Comment:
Agreed, but container level is bound to have a number of local state values, as well as logic behind setting them.